src/test/apiTest/resources/application.yaml

```diff
@@ -88,6 +88,17 @@ template:
           rule-92-yes: ${TSE_RESPONDENT_REPLY_TO_TRIB_TO_RES_RULE92_YES:f75316e7-29b4-49e1-b04e-02241f69b649}
           rule-92-no: ${TSE_RESPONDENT_REPLY_TO_TRIB_TO_RES_RULE92_NO:376fb37d-d563-42f2-89f0-3165a4777447}
         tribunal: ${TSE_RESPONDENT_REPLY_TO_TRIB_TO_TRIB:c2f72be8-541c-4bcc-b213-42557976dc39}
+    claimant-rep:
+      application:
+        claimant-rep-a: ${TSE_CLAIMANT_REP_TO_CLAIMANT_REP_TYPE_A_TEMPLATE_ID:7232eb51-89fc-4fdf-b0a1-2d6c3e28955e}
+        claimant-rep-b: ${TSE_CLAIMANT_REP_TO_CLAIMANT_REP_TYPE_B_TEMPLATE_ID:19812581-df03-4bd6-ad0f-907ba90c6603}
+        claimant-rep-c: ${TSE_CLAIMANT_REP_TO_CLAIMANT_REP_TYPE_C_TEMPLATE_ID:82744586-f034-4e4d-b5f6-b0aeb23bd582}
+        claimant-no: ${TSE_CLAIMANT_REP_TO_CLAIMANT_REP_NO_TEMPLATE_ID:a3621d2d-3cd8-4811-87e7-fcffa676dd3b}
+        respondent-type-a: ${TSE_CLAIMANT_REP_TO_RESPONDENT_TYPE_A_TEMPLATE_ID:410270f3-4136-4f3c-99d7-0a564955803f}
+        respondent-type-b: ${TSE_CLAIMANT_REP_TO_RESPONDENT_TYPE_B_TEMPLATE_ID:36161401-e4a2-4a98-925c-a65529d23fad}
+        cyRespondent-type-a: ${CY_TSE_CLAIMANT_REP_TO_RESPONDENT_TYPE_A_TEMPLATE_ID:3b3b3b3b-3b3b-3b3b-3b3b-3b3b3b3b3b3b}
+        cyRespondent-type-b: ${CY_TSE_CLAIMANT_REP_TO_RESPONDENT_TYPE_B_TEMPLATE_ID:3b3b3b3b-3b3b-3b3b-3b3b-3b3b3b3b3b3b}
```

Comment:
Are these two(cyRespondent-type-a and cyRespondent-type-b) using the same template?

Reply:
templates updated